agent/conf.go

```diff
@@ -19,6 +19,7 @@ type Conf struct {
 
 type FileConf struct {
 	Path    string `toml:"path"`
+	Command string `toml:"command"`
```

Comment:
NB this is a UI change, in that `command` means what `process` used to mean. I think the semantics are clearer with this name, but maybe I should leave `process` with the old semantics and pick a new name for the process watching config. Or even maybe we don't want a string match based implementation at all...